src/app/components/ad/ad.component.scss

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2021 Bitfly GmbH
```

Comment:
Minor: We might want to update this to 2023 in all files. Thoughts?

Reply:
Don't have a strong opinion on this. Best to update all at once in a separate PR though.